config/prow/plugins.yaml

```diff
@@ -767,13 +767,13 @@ require_matching_label:
   org: kubernetes-sigs
   repo: cluster-api-provider-aws
   issues: true
-  prs: false
+  prs: true
   regexp: ^priority/
 - missing_label: needs-triage
   org: kubernetes-sigs
   repo: cluster-api
   issues: true
-  prs: true
+  prs: false
```

Comment:
It's fine, but wasn't the goal of this PR to disable it for CAPA?

Reply:
This really is a mistake I think 😢 I will correct it in follow up PR.